ceph/rados/rados_bench.py

```diff
@@ -279,6 +280,41 @@ def continuous_run(self, client, pool_name, duration):
 
         return "Done....."
 
+    def radosBench_write(self, client, pool_name, time_in_sec):
```

Comment:
maybe redundant with write method.

Reply:
rados_bench.py file, The file contains rados bench related operations. In this file, the purpose of the "write"  method is to write the data in the cluster. While passing inputs to the method getting "illegal target for variable annotation" & "positional argument follows keyword argument".There is no clarity on how to pass the parameters.